platforms/stellar-disbursement-platform/admin-guide/30-getting-started.mdx

```diff
@@ -380,25 +404,21 @@ Please make sure to update the appropriate configuration on the Anchor Platform
 
 :::
 
-### Countries
-
-To customize wallets and countries, you’ll need to do some backend work.
-
-The list of available countries is seeded with a database patch. The default list includes every country except North Korea, Iran, Cuba, and Syria for easy testing. If you want to narrow the list, you will need to remove the country record directly from the countries table within the SDP database.
+#### Wallets
 
-Here is an example of how to remove France with its three-character ISO 3166 code:
-
-<CodeExample>
+For a full overview on how to add wallets and how to make a wallet SDP-compatible, check out the [Making Your Wallet SDP-Ready](making-your-wallet-sdp-ready) guide.
 
-```sql
-DELETE FROM countries WHERE code = 'FRA';
-```
+### Wallet Address Registration
 
-</CodeExample>
+Since version [`3.0.0`](https://github.com/stellar/stellar-disbursement-platform-backend/releases/tag/3.0.0), users can pay directly to Stellar wallet addresses rather than driving the users through the registration flow. This is useful for organizations that are paying receivers who already have Stellar wallets and don't need to create a new one.
```

Comment:
```suggestion
Since version [`3.0.0`](https://github.com/stellar/stellar-disbursement-platform-backend/releases/tag/3.0.0), users can pay directly to Stellar wallet addresses rather than directing the users through the registration flow. This is useful for organizations that are paying receivers who already have Stellar wallets and don't need to create a new one.
```